core/src/MegaMolGraph.cpp

```diff
@@ -267,8 +529,23 @@ bool megamol::core::MegaMolGraph::add_module(ModuleInstantiationRequest_t const&
 
     if (!isCreateOk) {
         this->module_list_.pop_front();
+    } else {
+        // iterate parameters, add hotkeys to CommandRegistry
+        for (auto child = module_ptr->ChildList_Begin(); child != module_ptr->ChildList_End(); ++child) {
+            auto ps = dynamic_cast<param::ParamSlot*>((*child).get());
+            if (ps != nullptr) {
+                auto p = ps->Param<param::ButtonParam>();
+                if (p != nullptr) {
+                    frontend_resources::Command c;
+                    c.key = p->GetKeyCode();
+                    c.parent = ps->FullName();
+                    c.name = module_ptr->Name().PeekBuffer() + std::string("_") + ps->Name().PeekBuffer();
+                    c.effect = this->Parameter_Lambda;
+                    m_command_registry->add_command(c);
```

Comment:
I think we can abstract this code away when we get the Graph subscriber / Graph diffs / Graph serialization infrastructure up.